src/Reader/Http/Response.php

```diff
@@ -0,0 +1,175 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Zend\Feed\Reader\Http;
+
+use Zend\Feed\Reader\Exception;
+
+class Response implements HeaderAwareResponseInterface
```

Comment:
@harikt — I've added `Psr7ResponseDecorator` to the pull request at this time; PSR-7-based clients can then do something like the following:

```php
return new Psr7ResponseDecorator($response);
```

within their `ClientInterface::get()` method in order to return something usable by `Reader`.

In order to facilitate this, and to remove ambiguity, I renamed `getHeader()` to `getHeaderLine()`, mirroring what's in PSR-7's `MessageInterface`.